<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MigrateDialog</class>
 <widget class="QDialog" name="MigrateDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>331</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ryzom Installer</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0,0,0">
   <item>
    <widget class="QLabel" name="messageLabel">
     <property name="text">
      <string>Welcome to Ryzom Installer!

This program will migrate Ryzom version 2.1 to 3.0. Your files will be updated, cleaned and moved to a new location.

Just press Continue button and follow the different steps until everything is done.</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignJustify|Qt::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="advancedLayout">
     <item>
      <spacer name="advancedHorizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="advancedCheckBox">
       <property name="text">
        <string>Show advanced parameters (expert)</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="advancedFrame">
     <layout class="QVBoxLayout" name="advancedMainLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>9</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QGroupBox" name="destinationGroupBox">
        <property name="title">
         <string notr="true">Files will be installed to (requires %1):</string>
        </property>
        <layout class="QHBoxLayout" name="destinationLayout" stretch="1,0,0">
         <item>
          <widget class="QLabel" name="destinationLabel">
           <property name="text">
            <string notr="true">c:\</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="destinationDefaultButton">
           <property name="text">
            <string>Default</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="destinationBrowseButton">
           <property name="text">
            <string>Browse...</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="clientArchGroupBox">
        <property name="title">
         <string>Do you prefer to use a 64 or 32 bit client?</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QRadioButton" name="clientArch64RadioButton">
           <property name="text">
            <string>64 bit (recommended)</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="clientArch32RadioButton">
           <property name="text">
            <string>32 bit</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonsLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="continueButton">
       <property name="text">
        <string>Continue</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="quitButton">
       <property name="text">
        <string>Quit</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>advancedCheckBox</tabstop>
  <tabstop>destinationDefaultButton</tabstop>
  <tabstop>destinationBrowseButton</tabstop>
  <tabstop>clientArch64RadioButton</tabstop>
  <tabstop>clientArch32RadioButton</tabstop>
  <tabstop>continueButton</tabstop>
  <tabstop>quitButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
